### DISCUSSION TITLE
How can I position a table?

### DISCUSSION THREAD

**Discussion by stevenlis** [py-pdf/fpdf2#988] [Q&A] [answered]
It appears that a table can only be positioned in the middle of a page.

**Comment by Lucas-C** [comment, marked as answer]
Hi @stevenlis

Have you checked the `align` parameter of `FPDF.table()`?
* https://py-pdf.github.io/fpdf2/fpdf/fpdf.html#fpdf.fpdf.FPDF.table
* https://py-pdf.github.io/fpdf2/fpdf/table.html#fpdf.table.Table

It's used there in the source code: https://github.com/py-pdf/fpdf2/blob/2.7.6/fpdf/table.py#L231

Do you require some other kind of positioning?

**Reply by stevenlis** [reply]
@Lucas-C Thanks, man. I didn't realize that it's the alignment of the table on a page and thought it's the alignment within cells. Do you have any plans to make those documents more modern, like the templates used in [PyData](https://pydata-sphinx-theme.readthedocs.io/en/stable/)? It's a bit difficult to search through the current version.

**Reply by stevenlis** [reply]
@Lucas-C Is there a way to control the font size of a table separately? It seems that `size_pt` is no longer working.

**Reply by Lucas-C** [reply]
> Do you have any plans to make those documents more modern, like the templates used in [PyData](https://pydata-sphinx-theme.readthedocs.io/en/stable/)? It's a bit difficult to search through the current version.

We have no such plans at the moment, but I'm open to improvements on our documentation!
I checked the website you linked to, but what templates are you refering to?
HTML templates ? _cf._ https://pydata-sphinx-theme.readthedocs.io/en/stable/user_guide/layout.html#templates-and-components

More broadly, what features are you looking for exactly?
Do you find the search engine on the documentation website of `fpdf2` limited in some way?

> Is there a way to control the font size of a table separately? It seems that size_pt is no longer working.

Yes.

The simplest way to do this would be to call `pdf.set_font(size=24)` before rendering a table.

You can also use `with pdf.local_context(font_size=16): ...`
_cf._ https://py-pdf.github.io/fpdf2/fpdf/fpdf.html#fpdf.fpdf.FPDF.local_context

**Reply by stevenlis** [reply]
@Lucas-C Thank you for providing the details. One example is the website https://pola-rs.github.io/polars/py-polars/html/reference/, where you can easily find all the APIs.

**Reply by Lucas-C** [reply]
> One example is the website [pola-rs.github.io/polars/py-polars/html/reference](https://pola-rs.github.io/polars/py-polars/html/reference/), where you can easily find all the APIs.

Thanks.
What do you like about this website?
The fact that all method names are listed on a single page?

**Reply by stevenlis** [reply]
It's easier to navigate. I struggle to understand the relationship between modules when reading the documentation at https://py-pdf.github.io/fpdf2/fpdf/. It's unclear which methods are available. There's no built-in search engine, but you can perform a page search. However, it may return hundreds of matches.

**Reply by Lucas-C** [reply]
Thank you for the clear feedback
